proposal/color-4-new-spaces-js.d.ts.md

```diff
@@ -495,8 +672,8 @@ constructor(
 
 ### Deprecations
 
-A number of SassColor getters only make sense for legacy colors, and so are
-being deprecated for `channel`.
+A number of SassColor getters only make sense for [legacy color space], and so
+are being deprecated for `channel`.
```

Comment:
We've started adding explicit deprecation names, since users can specify deprecations to mute or upgrade to errors in the API. How about `color-4-api` for these?